getdokan/dokan
# Tailwind css added

### All Submissions:






### Changes proposed in this Pull Request:

Added tailwindcss to dokan lite

Closes # .
* Dokan pro PR: https://github.com/getdokan/dokan-pro/pull/2467

### How to test the changes in this Pull Request:

1. Test all dokan menus if design breaks or changes
2. Check list table and table filter padding and margin
3. Check of section `At a glance` in dokan `Dashboard` menu
4. Check admin notice style
5. Check `whats new` icon and style



### Changelog entry

> Added tailwindcss to dokan lite


### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.


### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.
